cohortextractor/emis_backend.py

```diff
@@ -78,8 +78,9 @@ def to_csv(self, filename):
         with open(filename, "w", newline="") as csvfile:
             writer = csv.writer(csvfile)
             writer.writerow([x[0] for x in result.description])
-            for row in result:
+            for ix, row in enumerate(result):
                 unique_check.add(row[0])
+                row[0] = ix
```

Comment:
I think this is sensible, but I know there have been a couple of occasions where researchers have relied on patient IDs being consistent to join across extracts from multiple study definitions. The sequential ints here mean that if they try to do this the join will "work" in the sense that there'll be lots of matches, but the results will be nonsense. I wonder if generating a large random offset would help avoid this. Or is consistency between multiple extracts of the same study is desirable we could base the offset on a hash of the SQL or something like that.

Probably not worth worrying about in the immediate term, but we might need to revisit this at some point.

Reply:
Maybe we could do `row[0] = md5(row[0])` or something.  (@bloodearnest's original idea.)